src/main/java/com/uid2/admin/vertx/service/AdminKeyService.java

```diff
@@ -125,70 +130,67 @@ private void handleAdminMetadata(RoutingContext rc) {
                     .putHeader(HttpHeaders.CONTENT_TYPE, "application/json")
                     .end(adminUserProvider.getMetadata().encode());
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Don't do this. The service could be spammed.

Reply:
Relic of older debug code, thanks